src/pages/_app.tsx

```diff
@@ -142,6 +142,8 @@ App.getInitialProps = async ({ Component, ctx }) => {
 	let host
 	if (ctx.req) {
 		host = ctx.req.headers.host
+	} else {
+		host = window.location.host
```

Comment:
Fixing an issue where the host for the og-image tag was undefined during a clientside request.